### PR TITLE
Build rb-json

### PR DESCRIPTION
**Pull request by oblute**
Checklist

- [X ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X ] Source is from official source
- [X ] Package does not vend other packages
- [X ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by oblute** [comment]
@sodre this package is ready for your review.
@benhuff do you agree to be a maintainer?

File: recipes/rb-json/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 9dd1437156773f72c096058ec837faac1b00077121a3fd574e68f895ea3aa96b
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }} clean
```
Comment (sodre) [inline recipes/rb-json/meta.yaml]: This gem extensions in sub-directories to ext that need to be cleaned up. You can see these from the `patchelf: wrong ELF type` warnings in the [linux build logs][linuxlogs].
<details>
  <summary>build logs snippet: lines [616,625]</summary>

```
Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
patchelf: wrong ELF type
patchelf: --print-rpath failed for /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/ruby/gems/2.6.0/gems/json-2.2.0/ext/json/ext/parser/parser.o

Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
patchelf: wrong ELF type
patchelf: --print-rpath failed for /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/ruby/gems/2.6.0/gems/json-2.2.0/ext/json/ext/generator/generator.o
```
</details>

```suggestion
    {% for dir in [ '', 'ext/generator', 'ext/parser' ] %}
    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }}/{{ dir }} clean
    {% endfor%}    
```

[linuxlogs]: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=68987&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=616&lineEnd=625&colStart=1&colEnd=1

File: recipes/rb-json/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  skip: {{ win }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - ruby
+    - openssl
```
Comment (sodre) [inline recipes/rb-json/meta.yaml]: It looks like `openssl` is not a requirement from the library according to the [build logs][linuxlogs] either. Please try building without it if you haven't tried it yet, o.w. just let me know.
<details>
  <summary>build logs snippet: lines [640,641]</summary>

```
WARNING (rb-json): run-exports library package conda-forge::openssl-1.1.1a-h14c3975_1001 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```
</details>

```suggestion
```

[linuxlogs]: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=68987&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=640&lineEnd=641&colStart=1&colEnd=1

**Comment by oblute** [comment]
@sodre thanks! I made those two changes.

**Pull request merged**